kclvm/driver/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+use kclvm_config::modfile::KCL_MOD_PATH_ENV;
+use kclvm_runtime::PanicInfo;
+use kclvm_utils::path::PathPrefix;
+use std::path::Path;
+
+/// Replace `${KCL_MOD}` in k_files with `work_dir`
```

Comment:
/// Normalize input files with the working directory and replace `${KCL_MOD}` with the module root path.